### ISSUE TITLE
[BUG] result.show_in_window() returns error 

### ISSUE THREAD

**Issue by TheSolY** [deepchecks/deepchecks#1917]
code to reproduce (from feature_label_correlation_test)
`

    def util_generate_second_similar_dataframe_and_expected():     
      np.random.seed(42)
      df, _ = util_generate_dataframe_and_expected()
      df2 = df.copy()
      df2['x2'] = df['x2'] + 0.5 * df['x1']
      df2['x3'] = 0.3 * df['x3'] + df['label']
      return df, df2, {'x1': close_to(0, 0.1), 'x2': close_to(0.3, 0.1), 'x3': close_to(-0.54, 0.1),
                       'x4': close_to(0, 0.1), 'x5': close_to(0, 0.1)}
    df, df2, expected = util_generate_second_similar_dataframe_and_expected()
    result = FeatureLabelCorrelationChange(n_show_top=3, random_state=42).run(
    train_dataset=Dataset(df, label='label'), test_dataset=Dataset(df2, label='label'))
    result.show_in_window()
`

error message:

![Screen Shot 2022-08-17 at 12 38 59](https://user-images.githubusercontent.com/99395146/185088090-5ee78470-9c96-4425-9fce-1d28ae9604bb.png)

**Comment by yromanyshyn** [comment]
NOTE:

this happens only on platforms that do not support or do not use by default `fork` process creation method (macOS, Windows)

**Comment by yromanyshyn** [comment]
@ItayGabbay it is not completed yet,
@TheSolY tried on macOS and it failed
